rewrite-groovy/src/test/java/org/openrewrite/groovy/tree/CastTest.java

```diff
@@ -73,7 +73,7 @@ void groovyCastAndInvokeMethod() {
         rewriteRun(
           groovy(
             """
-              ( "" as String ).toString()
+              (((((( "" as String ))))).toString())
```

Comment:
also here: perhaps best to keep both the original and the unexpected case you've modified for your purposes.